strings.js

```diff
@@ -0,0 +1,263 @@
+//         Hint
+//     </summary>
+//     It should work for strings of any length.
+// </details>
+//
+// Examples, for variables:
+// ```js
+// let name1 = 'Taq' // Should log 'Taq'
+// let sentence = 'He is Taq'// Should log 'Taq'
+// ```
+
+let threat = "You need to back off before I eliminate you";
+if (threat.length < 3) {
+  console.log("#ERROR#");
+} else if (threat.length >= 3) {
+  console.log(threat.slice(threat.length - 3));
+}
+
+let threat2 = "Hi";
+if (threat2.length < 3) {
+  console.log("#ERROR#");
+} else if (threat2.length >= 3) {
+  console.log(threat2.slice(threat2.length - 3));
+}
+
+//
+// 3. Write a code block that prints/logs whether a string `x` starts with a
+// vowel.
+// <details>
+//   <summary>
+//     Extra
+//   </summary>
+//
+//   Make it [case insensitive](https://en.wikipedia.org/wiki/Case_sensitivity)
+//
+// </details>
+//
+
+let stringX = "all of you are fired!";
+if (stringX[0] === "A") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "E") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "I") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "O") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "U") {
+  console.log("This string starts with a vowel.");
+} else if (stringX[0] === "a") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "e") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "i") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "o") {
+  console.log("This string starts with a lowercase vowel.");
+} else if (stringX[0] === "u") {
+  console.log("This string starts with a lowercase vowel.");
+}
+
+//
+// 4. Write a code block that given two strings, `str1` and `str2`,
+// prints/logs a new string with `str2` mashed into the middle of `str1`
+//
+// Examples:
+//
+//   - For `str1` = 'help' and `str2` = 'me'. Your program should log `'hemelp'`
+//
+//    - For `str1` = 'hello' and `str2` = 'world'. Your program should log
+//    `'helworldlo'`
+//
+
+let str1 = "hello";
+let str2 = "world";
+console.log(str1.slice(0, 3) + str2 + str1.slice(3));
+
+//
+// 5. Write a code block that prints/logs whether a string `x` has at least one
+// vowel in it.
+//
+// Examples:
+//   - For `x` = 'taq'. Your program should log `'taq' has a vowel'`
+//   - For `x` = 'dfghjkl'. Your program should log `'dfghjkl' does not have
+//   a vowel'`
+//   - For `x` = '' Your program should log `'' does not have a vowel'`
+//
+// <details>
+//   <summary>
+//     Extra & Hint:
+//   </summary>
+//
+//   - **Hint**: Abstain from using loops. Explore string methods.
+//
+//   - **Extra**: Make it [case insensitive]
+//   (https://en.wikipedia.org/wiki/Case_sensitivity). So that for
+//   `x` = 'TAQ' it should log `'TAQ' has a vowel`
+// </details>
+//
+
+let word = "Rhythm";
```

Comment:
Another quick way of solving this is using Regex. Although they tend to look like some cat walked on the keyboard, they are useful in making code shorter and potentially more readable.